### PR TITLE
fix(site): resize terminal when dismissing warning

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

File: site/src/pages/TerminalPage/TerminalPage.tsx [hunk 1]
```diff
@@ -29,6 +29,46 @@ export const Language = {
...
+    } else if (lifecycleState === "starting") {
+      setStartupWarning("starting")
+    } else {
+      setStartupWarning((prev) => {
+        if (prev === "starting") {
+          return "success"
+        }
+        return undefined
+      })
+    }
```
Comment (Kira-Pilot) [inline site/src/pages/TerminalPage/TerminalPage.tsx:57]: copilot?

**Review by Kira-Pilot — APPROVED** [review]

File: site/src/pages/TerminalPage/TerminalPage.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/pages/TerminalPage/TerminalPage.tsx:57]: Maybe? I just moved the existent code.

**Comment by mafredri** [comment]
@BrunoQuaresma I noticed the issue that (I think?) this was supposed to address is still present (from dev.coder.com, today):

Before dismiss:

image

After (text should go to the bottom but it's permanently "in the air" until I resize the browser window):

image

**Comment by BrunoQuaresma** [comment]
Strange.... can you please re-open the issue and assign it to me?

**Comment by mafredri** [comment]
@BrunoQuaresma I couldn't find one, so I opened a new one https://github.com/coder/coder/issues/8175

**Pull request merged**